main.rb

```diff
@@ -1,12 +1,27 @@
-# main.rb
-
-require_relative 'person'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
-
-person = Person.new(22, 'maximilianus')
-person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+#!/usr/bin/env ruby
+
+require_relative 'app'
+require_relative 'menu'
+require_relative 'actions'
+
+def main
+  welcome_message
+
+  loop do
+    display_options
+
+    option = gets.chomp.to_i
+
+    case option
+    when 1 then create_student
+    when 2 then create_teacher
+    when 4 then create_book
+    when 5 then create_rental
+    when 6 then list_rentals_for_person
+    when 7 then break
```

Comment:
- [ Required ] I think it would be nice if the `when` clauses of your switch statements match with the options provided in the `display_options` method in your project. Currently, when I enter option `3` the application responds with the message _Invalid option. Please try again._ yet when I choose option `4` which is `create a rental` it prompts me to create a book. This shows that the `options` in the menu don't match with the choices in the `switch` statement. I think the options should be as shown below.

## Example code
```ruby
case option
    when 1 then create_student
    when 2 then create_teacher
    when 3 then create_book
    when 4 then create_rental
    when 5 then list_all_books
    when 6 then list_all_people
    when 7 then list_rentals_for_person
    when 8 then break
```

I was neither able to create a book so I could not test for `listing books`, `renting a book`, or `listing rentals`, please make sure these too work.